code/modules/projectiles/bullet.dm

```diff
@@ -1914,6 +1910,36 @@ ABSTRACT_TYPE(/datum/projectile/bullet/homing/rocket)
 		. = ..()
 		return . && isliving(M) && !isintangible(M)
 
+/datum/projectile/bullet/homing/rocket/salvo
+	name = "Salvo Rocket"
+	max_rotation_rate = 5
+	dissipation_delay = 30
+	start_speed = 15
+	explosion_power = 1
+	shot_delay = 0.3 SECONDS
+	var/initial_projectile = TRUE
+
+	on_launch(obj/projectile/P)
+		..()
+		if (!P)
+			return
+		if (!src.initial_projectile)
+			return
+		P.rotateDirection(rand(-30, 30))
+		SPAWN(0)
+			for (var/i in 1 to 2)
+				sleep(src.shot_delay)
+				var/datum/projectile/bullet/homing/rocket/salvo/rocket = new()
+				rocket.initial_projectile = FALSE
+				var/turf/T = get_turf(P.shooter)
+				var/obj/projectile/new_rocket = initialize_projectile(T, rocket, P.xo, P.yo, P.shooter)
```

Comment:
does not play nicely with large pods - the additional projectiles will all come out of the bottom-left corner of the pod, instead of on each tile in front of the pod's facing direction